reminders-menubar/Resources/remindersLocalized.swift

```diff
@@ -105,7 +105,8 @@ func rmbCurrentLocale() -> Locale {
     if let preferredLanguage = UserPreferences.shared.preferredLanguage {
         currentLocale = Locale(identifier: preferredLanguage)
     }
-    if Bundle.main.path(forResource: currentLocale.identifier, ofType: "lproj") == nil {
+    let Has = Bundle.main.localizations.contains
+    if !(Has(currentLocale.identifier) || Has(currentLocale.languageCode ?? "")) {
```

Comment:
small change to make it clearer what the variable means and to eliminate the linter error about variable starting with a capital letter.

```suggestion
    let containsLocalization = Bundle.main.localizations.contains
    if !(containsLocalization(currentLocale.identifier) || containsLocalization(currentLocale.languageCode ?? "")) {
```

Reply:
Don't worry about the linter, this project is using [SwiftLint](https://github.com/realm/SwiftLint), but it's not a blocker not using it.

Within a small scope, using a short name (like just a verb) can be ok, but even so, trying a name that makes it more explicit is usually more recommended. You can check, for example, that in line 107 I'm using `var currentLocale = Locale.current` and not just `var current = Locale.current`.

Taking a closer look at the results of your changes, I think a more interesting approach would be to not have a fallback for en_US.

What is happening for en_GB is that en_GB is being used everywhere that `rmbCurrentLocale()` is called (date pickers, date formatters and relative date formatting). But when using the translated strings for the app, a fallback to en_US is occurring (within the `rmbLocalized` function in `let localeBundle = Bundle(path: localePath) ?? Bundle.main`).

So I think I'll change it so that whenever "use system language" is enabled, the system language will be used whenever possible. Meaning that for a language that is not in the app, all strings will be in en_US, but the date picker, date formatters and relative date formatting will be in the system language.

Let me know what you think of this approach. I will add it as a suggestion for the code.

Reply:
Ah! I had assumed because it was `en.lproj` that whatever Swift magic was at work (mind my naïvete in the language) auto-resolved both `en_gb` and `en_us` to use these strings, and that by dint of there being no `en_us.lproj` bundle both locales would silently follow the code path.

Thanks for the suggestion!